### DISCUSSION TITLE
githubusercontent.com is not replying

### DISCUSSION THREAD

**Discussion by johnerinjery** [ScoopInstaller/Main#4252] [Q&A]
I'm trying to install dark on my device when this error popped up:

**A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond <an address>
URL https://raw.githubusercontent.com/ScoopInstaller/Binary/master/dark/dark-3.11.2.zip is not valid**

I tried pinging the website at raw.githubusercontent.com only to not get a reply.

**Comment by rashil2000** [comment]
Can you access the link through the browser?

**Reply by johnerinjery** [reply]
sorry for the late reply. No i am not able to access the site. it is not connecting i tried resetting my internet as well. no effect

**Reply by rashil2000** [reply]
You'll have to look at your internet settings. Or github might be geoblocked in your location.

**Comment by rasa** [comment]
You might try https://github.com/ScoopInstaller/Binary/raw/master/dark/dark-3.11.2.zip . That's another way to access that file.
